postgres/src/test/scala/zio/sql/postgresql/FunctionDefSpec.scala

```diff
@@ -854,8 +872,6 @@ object FunctionDefSpec extends PostgresRunnableSpec with ShopSchema {
       } yield assert(r.head)(equalTo("UTF8"))
 
       assertion.mapErrorCause(cause => Cause.stackless(cause.untraced))
-    }
-  )
-
-  private def postgresStringEscape(s: String): String = s""" '${s}' """
+    } @@ ignore  //todo fix - select(PgClientEncoding())?
```

Comment:
`from smth` is required now. We need to rethink `Read`.

Reply:
I raised this #323 for this one. I think we should rename `Read.Select` to `Read.SelectFrom` and `SelectBuilder` to `Read.Select`, both extending a common trait which the renderer can match on.

I think the underlying issue with this one I think is just that it needs parens and other `Function0`s don't.

This is one of the changes I was talking about in my comment on https://github.com/zio/zio-sql/pull/290#discussion_r531206389